packages/authenticator/amplify_authenticator/lib/src/state/authenticator_state.dart

```diff
@@ -139,7 +139,46 @@ class AuthenticatorState extends ChangeNotifier {
     notifyListeners();
   }
 
-  String _confirmationCode = '';
+  late String _confirmationCode;
```

Comment:
Not sure I follow this - it's reset to `''` below anyway and is read as the initial value on many screens.

Reply:
Good catch, this wasn't meant to be changed.